examples/experimental/dagster-airlift/examples/mwaa-example/scripts/setup_mwaa.py

```diff
@@ -0,0 +1,120 @@
+"""This script uploads DAG files and requirements.txt to an S3 bucket in a directory
+matching the name of the specified MWAA environment. It then attempts to update
+the MWAA environment with the new S3 location. If the environment doesn't exist,
+it provides instructions for creating one.
+
+The script prompts for user confirmation before deleting any existing contents
+in the target S3 directory, then uploads the new files, ensuring a clean deployment.
+"""
+
+import os
+from typing import List
+
+import boto3
+import click
+from botocore.exceptions import ClientError
+
+
+def get_s3_objects(s3_client: boto3.client, bucket: str, prefix: str) -> List[dict]:
+    """Get list of objects in the S3 bucket with the given prefix."""
+    try:
```

Comment:
This is a lot of code. Does MWAA not provide some of this stuff as utilities? This looks like we are reimplementing AWS tooling with ChatGPT or something.

Reply:
I ripped this straight from the aws documentation; didn't use chat GPT for this